packages/forms/__mocks__/data.js

```diff
@@ -40,6 +40,65 @@ export const data = {
 	errors: {},
 };
 
+export const nestedData = {
```

Comment:
I don't think you need that complicated schema to test an id separator. Can you put the minimum amount of schema please ? The easier it is, the more it is maintainable.

Reply:
> Why is it `annoying when the key property path contains values with _ within it` ? What is the issue behind ?

if fails to split id and replace to work with a new wrapper in TPD

Reply:
> I don't think you need that complicated schema to test an id separator. Can you put the minimum amount of schema please ? The easier it is, the more it is maintainable.

I don't get what complexity you are talking about, it is very similar to the data var that is in place

Reply:
You need data for your test. Your test only check the timestamp part. But the data is a schema describing jsonSchema and uiSchema of timestamp, firstname, and lastname.

>  it is very similar to the data var that is in place
That's not the point, I haven't seen how the other data are used, but that doesn't mean it's good to do the same here.